README.md

```diff
@@ -66,6 +66,14 @@ $ make quake
 
 The usage and limitations of Doom and Quake demo are listed in [docs/demo.md](docs/demo.md).
 
+### Docker image
+
+The image containing all the necessary tools for development and testing can be built by executing `docker build -t rv32emu_ubuntu .`. 
```

Comment:
Why did you refer it as `rv32emu_ubuntu`?

Reply:
Oops sorry, it should be just `rv32emu`